core/src/main/java/io/temporal/samples/nexuscancellation/README.MD

```diff
@@ -2,9 +2,10 @@
 
 This sample shows how to cancel a Nexus operation from a caller workflow.
 
-From more details on Nexus and how to setup to run this samples please see the [Nexus Sample](../nexus/README.MD).
+Follow the instructions in the main [Nexus Sample](../nexus/README.MD) to set up your environment to run this sample
```

Comment:
Maybe a bit pedantic/nit, but does this read better to you:
`To run this sample, setup your environment following the instructions in the main [Nexus Sample](../nexus/README.md).`

Reply:
Yes, thanks, taken! Although as a fellow pedant I am far from convinced that "setup" is a verb!